internal/dispatch/keys/hasher_common.go

```diff
@@ -111,14 +110,13 @@ func (hsv hashableStructValue) AppendToHash(hasher hasherInterface) {
 		hasher.WriteString("null")
 
 	case *structpb.Value_NumberValue:
-		hasher.WriteString(fmt.Sprintf("%f", t.NumberValue))
+		// AFAICT, this is how Sprintf-style formats float64s
```

Comment:
So long as it produces a valid and unique hash, doesn't really matter the exact form